### DISCUSSION TITLE
[minimal-blog]: Make the dropdown menu for page

### DISCUSSION THREAD

**Discussion by muhammadizzuddin** [LekoArts/gatsby-themes#610] [Questions] [answered]
## Summary

Its amazing template for me, but when its show in mobile device the menu seem crowded, can you make dropdown menu for it?

### Basic example

If the proposal involves a new or changed API, include a basic code example. Omit this section if it's not applicable.

### Motivation

Why are we doing this? What use cases does it support? What is the expected outcome?
We ussualy read everything at our mobile device, its make reader more comfortable because its look nice in dropdown menu

**Comment by LekoArts** [comment, marked as answer]
Hi!

I don't plan on adding a dropdown to the theme but you can implement this yourself by [shadowing the theme](https://github.com/LekoArts/gatsby-themes/tree/master/themes/gatsby-theme-minimal-blog#shadowing), e.g. the [navigation](https://github.com/LekoArts/gatsby-themes/blob/master/themes/gatsby-theme-minimal-blog/src/components/navigation.tsx) or [header](https://github.com/LekoArts/gatsby-themes/blob/master/themes/gatsby-theme-minimal-blog/src/components/header.tsx)
